DISCLAIMER

```diff
@@ -1,7 +0,0 @@
-Apache Wayang is an effort undergoing incubation at The Apache Software
```

Comment:
We should keep this, since it's still an incubating project.

Reply:
Ah was purged as I cleaned the directory. Yes, agreed, great review @suyanhanx!